bitbybit/Cargo.toml

```diff
@@ -18,7 +18,7 @@ proc-macro = true
 experimental_builder_syntax = []
 
 [dependencies]
-syn = "2.0"
```

Comment:
why this change?

the current version of proc-macro2 for example is "1.0.69", so this change would move us back in time

Reply:
That's an error on my part.

Context: The third party crates.io registry called `lib.rs` has a "lint" page on dependencies (see yours here <https://lib.rs/~danlehmann/dash>). I found on my own lint page that it is better to specify the whole version string, due to potential violation of semantic versioning (there are at least two well known cases in the rust ecosystem).

But, at the time, I was specifying just the major version number, not even the minor one.

I think you are right, specifying the patch number is superfluous.